src/hooks/useSelector.js

```diff
@@ -31,7 +32,7 @@ function useSelectorWithStoreAndSubscription(
       selector !== latestSelector.current ||
       latestSubscriptionCallbackError.current
     ) {
-      selectedState = selector(store.getState())
+      selectedState = selector(latestReduxState.current || reduxState)
```

Comment:
Is this safe?
Imagine store state is a.. string:
```const reducer = (state = '', action) => { ... }```
If it's a falsy value `0 || null || undefined || ''`, then it'd always use `reduxState` and not `latestReduxState.current`. In most cases, this shouldn't really happen since it's unusual to have a store that holds a single value but still ? 😛 (same in connect)

Reply:
Yeah, this seems like a bug.

Reply:
yeah fixed https://github.com/reduxjs/react-redux/pull/1455/commits/eb6f8cf1742a6a319dac4a459e5513a1e6e50c7e